lib/OpenQA/Downloader.pm

```diff
@@ -82,8 +82,9 @@ sub _get {
     }
 
     if (!$res->is_success) {
-        my $message = $res->error->{message};
-        $log->info(qq{Download of "$target" failed: $code $message});
+        my $error   = $res->error;
+        my $message = ref $error eq 'HASH' ? " $error->{message}" : '';
```

Comment:
Maybe a default message would be nice, e.g. "unknown".
I wonder what kind of error would result in no message.

Reply:
Don't we at least get a code then? :thinking:

Reply:
Yeah, there is a code. I think adding an explicit hint that the message was empty would be helpful. But not important.